doc/mason-lspconfig-mapping.txt

```diff
@@ -202,4 +203,4 @@ yaml-language-server                      yamlls
 zk                                        zk
 zls                                       zls
 
-vim:tw=78:ft=help:norl:expandtab:sw=4
\ No newline at end of file
+vim:tw=78:ft=help:norl:expandtab:sw=4
```

Comment:
I don't really understand why there is diff. Seems that original file doesn't add newline `\n` at the end.